README.md

```diff
@@ -43,6 +43,16 @@ java -Xmx100G -jar "build/libs/GitHubWrapper-1.0-SNAPSHOT.jar" \
 - Using the `-repo` parameter, you specify the file path of the repo you want to analyze. Notice that you need to have cloned the repo locally, such that the origin can be derived from this file path.
 - Using the `-workDir` parameter, you specify the working directory, which usually is the directory which contains the repository directory specified at `-repo`.
 
+### `Referenced` events
+
+`Referenced` events are events generated in an issue if a commit references that issue in its commit message. The intended behavior is, that the event is present in the issue's event data, and the commit is again present in the related commits of the issue. This does not work if it is not possible to fetch that commit. In this case, the event still exists, but it contains a link to a commit that the api cannot resolve, meaning that no data about the commit can be accessed. This may lead to incorrect data points if the resulting data is automatically processed, for example using the tool `codeface-extraction`. Known causes of this include:
```

Comment:
"The intended behavior is, that " - no comma before "that".

---

"This may lead to incorrect data points if the resulting data is automatically processed, for example using the tool `codeface-extraction`." This needs more explanation: what are "incorrect data points"? Maybe you could give an example. And if you link to codeface-extraction, then I would link to the issue analysis script there.  Not sure if this is the correct place here to link to it. Maybe we move the section on referenced events to the end and have a separate section before where we link to codeface-extraction and explain that the data could further be processed using this tool? Otherwise a user of the GitHubWrapper might not understand what the relationship with codeface-extraction is and why this does matter...